src/main/resources/com/google/api/codegen/nodejs/method_sample.snip

```diff
@@ -45,7 +45,7 @@
 
 @private sampleWriteStreamingRequest(apiMethod)
     // Write request objects.
-    stream.write({@sampleMethodCallArgList(apiMethod)});
+    stream.write(request);
```

Comment:
It seems this is fine for now, at least this `request` variable name already appears twice in `nodejs/method_sample.snip`.  I'm wondering if this could conflict with other names in the sample potentially, but that might be a different topic.